test/services/notifications/setup/submit-check.service.test.js

```diff
@@ -74,6 +74,12 @@ describe('Notifications Setup - Submit Check service', () => {
     delete global.GlobalNotifier
   })
 
+  it('correctly returns the event id ', async () => {
```

Comment:
I will improve these test in another change. 

And use 'real' data with creating events / notifications etc.